analyzer/protocol/ipsec/main.zeek

```diff
@@ -672,8 +672,8 @@ function set_session(c: connection)
 event protocol_confirmation(c: connection, atype: Analyzer::Tag, aid: count) &priority=5
 	{
 	if ( atype == Analyzer::ANALYZER_SPICY_IPSEC_IKE_UDP ||
-	     atype == Analyzer::ANALYZER_SPICY_IPSEC_TCP ||
-	     atype == Analyzer::ANALYZER_SPICY_IPSEC_UDP )
+		 atype == Analyzer::ANALYZER_SPICY_IPSEC_TCP ||
```

Comment:
FWIW, I think this we'd actually like to keep: we indent this line with a tab like the `if` on the line above, and then do manual aligning with spaces. Indenting with tabs here will give inconsistent results depending on a user's tabwith.

Reply:
Switched to spaces.